akka-http-core/src/main/java/akka/http/javadsl/model/HttpMessage.java

```diff
@@ -20,6 +21,7 @@
  *
  * Binary compatibility is only maintained for callers of this trait’s interface.
  */
+@InternalApi
 public interface HttpMessage {
```

Comment:
I will replace `@InternalApi` with `@DoNotInherit` for model classes, except the ones that are `final case class` which are already not inheritable (these could keep the `@InternalApi`)